packages/demo-gatsby/gatsby-browser.js

```diff
@@ -19,13 +19,11 @@ distributed under the License is distributed on an "AS IS" BASIS,
 // custom typefaces
 import "typeface-montserrat"
 import "typeface-merriweather"
-import {Tina} from "tinacms";
-import * as React from "react";
-
-export function wrapRootElement({element}) {
-    return (
-        <Tina cms={window.tinacms}>
-            {element}
-        </Tina>
-    )
+import * as React from "react"
+import { TinaProvider, TinaCMS } from "tinacms"
+
+const cms = new TinaCMS()
```

Comment:
No need to create that instance
```suggestion
import { TinaProvider } from "tinacms"
```

Reply:
I wasn't sure 😊, thanks for the review.